docs/bulk_data.rst

```diff
@@ -173,6 +173,30 @@ the same Record Type upon load.
 It's recommended that new datasets use Record Type mapping by including the ``RecordTypeId`` 
 field. Using ``record_type`` will result in CumulusCI issuing a warning.
 
+Relative Dates
```

Comment:
@davidmreed , can you add a sample mapping that uses relative dates?   So, as a customer, I am confident I am setting up my mapping correctly.

Reply:
Yep. Duly added.